src/Core/src/Hosting/AppHostBuilder.cs

```diff
@@ -66,10 +71,6 @@ public IHost Build()
 
 			BuildFontRegistrar(_serviceProvider);
 
-			//we do this here because we can't inject the provider on the App ctor
-			//before we register the user ConfigureServices should this live in IApp ?
-			_app?.SetServiceProvider(_serviceProvider);
```

Comment:
This was always a bit weird in here. Now we don't need it as the host will set it once created.